openedx/openedx-platform
# Move content_tagging django app so that CMS-only tests run

## Description

This PR moves the content_tagging from `openedx/features/` to `openedx/core/djangoapps/content_tagging`  the tagging REST API for creating/updating object tags in Courses and XBlocks.

## Supporting Information

* More info on: https://github.com/openedx/edx-platform/pull/32907#discussion_r1312120204

## Testing instructions
* Ensure that the tests cover content_tagging in the `openedx-4` shard.
___
Private ref: [FAL-3500](https://tasks.opencraft.com/browse/FAL-3500)
